react_practice/react-apps/waitlist-page-v1/src/CTAForm.css

```diff
@@ -0,0 +1,58 @@
+.cta-form-btn {
+  display: block;
+  width: 50%;
+  margin: 10px auto 0;
```

Comment:
A couple things with this bit of code. If you look at the changes further down, you'll notice I removed the [`margin`](https://github.com/tyreljackson-L904/React_Dev/pull/1/files#diff-01af19101926742b006e1627be164446015739150c1f8d2572ceb0ede48d261fL10) and the [`max-width`](https://github.com/tyreljackson-L904/React_Dev/pull/1/files#diff-01af19101926742b006e1627be164446015739150c1f8d2572ceb0ede48d261fL7) properties. One of the best ways I found to make components more reusable is to let the parent component decide how best to place the child component in its view, which means that the child component should do its best not to try and lay itself out inside its view. `margin` is one of the absolute killers of this particular paradigm because it adds space _outside_ the component. By moving the `margin` here instead of the `PrimaryBtn`, it allows the parent to position the button exactly where it needs to go. If you need to use `PrimaryBtn` somewhere else, then it gives you a lot more flexibility on where to layout that component without needing to override a bunch of properties.